website/assets/sass/_pages.scss

```diff
@@ -1046,30 +1046,38 @@ margin-bottom: 3rem;
     font-weight: 600;
   }
   h3 {
-      border-bottom: 2px solid $slate-grey;
-      margin-bottom: .5rem;
-      width: 100%;
+    border-bottom: 2px solid $slate-grey;
+    margin-bottom: .5rem;
+    width: 100%;
   }
   .row {
     padding: 0 2rem;
   }
 }
 
+.related-power-data {
+  @extend %related-block;
+}
+
+.related-transmission-data {
```

Comment:
This needs some sort of whitespace on the bottom:

![image](https://user-images.githubusercontent.com/1625749/53445926-b04f7b80-39df-11e9-90a0-2d9f04e8857f.png)

The top whitespace is produced by the margin on the `h2` element … my feeling is that it shouldn't be and that the container itself should set a `padding` value that determines that outer whitespace. Changing that may disturb other `h2` elements elsewhere, and adding a container-specific rule may not be worth it either. Therefore setting a `margin-bottom` on the final element in the container may be the way to go.